src/analysis/normed_space/basic.lean

```diff
@@ -310,11 +318,24 @@ by simp only [inv_eq_one_div, norm_div, norm_one]
            ... = ∥a∥*∥a^k∥ : by rw normed_field.norm_mul
            ... = ∥a∥ ^ (k + 1) : by rw normed_field.norm_pow; simp [pow, monoid.pow]
 
+lemma nondiscrete_normed_field.has_small_element (α : Type*) [nnf : nondiscrete_normed_field α] :
```

Comment:
This is already in #829, see https://github.com/sgouezel/mathlib/blob/1d951c3960f6265a2e15275e59439ce46f41c188/src/analysis/normed_space/basic.lean#L342 (but maybe your version is better).

Reply:
Oups, I missed that. I think having x ≠ 0 is more convenient than ∥x∥ ≠ 0. Your name is mathlib-ier though. I'll adapt this once #829 is merged.

Reply:
I just merged #829.

Reply:
I've updated the PR